apps/vth-frontend/src/styles/globals.css

```diff
@@ -153,6 +153,35 @@ a {
   max-width: var(--utrecht-page-content-max-width);
 }
 
+@media screen and (-ms-high-contrast: active), screen and (forced-colors: active) {
+  .utrecht-accordion__header .utrecht-accordion__button {
+    border: 2px solid white;
+    max-inline-size: 100%;
+    position: relative;
+  }
+
+  .utrecht-accordion__header .utrecht-accordion__button::after {
+    background-image: var(
+      --utrecht-accordion-icon-arrow-down,
+      url("data:image/svg+xml;charset=utf-8,%3Csvg xmlns='http://www.w3.org/2000/svg' width='21.39' height='14.39'%3E%3Cpath fill='none' stroke='black' stroke-width='3' stroke-miterlimit='10' d='m15.866 4.135-5.32 5.322-5.322-5.322'/%3E%3C/svg%3E")
+    ) !important;
+    border-bottom: 2px solid white;
+    border-left: 2px solid white;
+    position: absolute;
+    right: -2px;
+    top: -2px;
+  }
+
+  @media (prefers-color-scheme: dark) {
+    .utrecht-accordion__header .utrecht-accordion__button::after {
```

Comment:
Misschien kunnen we de icon refactoren naar een `<svg>` in de HTML, en dan `fill="currentColor"` gebruiken. Dan hebben we maar 1 SVG nodig en dan kan de CSS veel simpeler blijven.

Reply:
Kun je me aanraden de steps om dat te doen. Ik ben niet zeker waar te beginnen. Hoe kan ik de html code van de accordion modificeren vanuit deze repository?